src/js/components/Pagination/Pagination.js

```diff
@@ -31,9 +31,23 @@ const Pagination = forwardRef(
     ref,
   ) => {
     const theme = useContext(ThemeContext) || defaultProps.theme;
+
+    let show;
+    let showItem;
+    // if children are provided, show can take form of { item: # },
+    // where # is the item to show
+    if (typeof showProp === 'object' && 'item' in showProp) {
```

Comment:
What if I pass `show={{page: 3}}`? Should we handle that case? `show = showProp.page`

Reply:
I like the option of handling both. I'll update.

Reply:
Done.